dotCMS/src/main/webapp/html/portlet/ext/workflows/view_workflow_task.jsp

```diff
@@ -499,19 +499,4 @@ public String getPostedby(String postedBy){
 
 <div dojoAttachPoint="fileBrowser" jsId="fileBrowser" onFileSelected="attachFileCallback" onlyFiles="true" dojoType="dotcms.dijit.FileBrowserDialog">
 </div>
-
-<form id="submitWorkflowTaskFrm" action="/DotAjaxDirector/com.dotmarketing.portlets.workflows.ajax.WfTaskAjax?cmd=executeAction">
```

Comment:
removing reference of the "old" form after the integration of the PP modals, https://github.com/dotcms/core/issues/15074